assets/wizards/advertising/views/ad-units/index.js

```diff
@@ -5,46 +5,46 @@
 /**
  * WordPress dependencies
  */
-import { Component, Fragment } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import { ActionCard, withWizardScreen } from '../../../../components/src';
+import { ActionCard, Router, withWizardScreen } from '../../../../components/src';
+
+/**
+ * Router component for managing single-page app nav.
+ */
+const { useHistory } = Router;
 
 /**
  * Advertising management screen.
  */
-class AdUnits extends Component {
-	/**
-	 * Render.
-	 */
-	render() {
-		const { adUnits, onDelete, service } = this.props;
+const AdUnits = ( { adUnits, onDelete, service } ) => {
+	const history = useHistory();
 
-		return (
-			<Fragment>
-				<p>
-					{ __(
-						'Set up multiple ad units to use on your homepage, articles and other places throughout your site. You can place ads through our Newspack Ad Block in the Editor.'
-					) }
-				</p>
-				{ Object.values( adUnits ).map( ( { id, name } ) => {
-					return (
-						<ActionCard
-							key={ id }
-							title={ name }
-							actionText={ __( 'Edit' ) }
-							href={ `#${ service }/${ id }` }
-							secondaryActionText={ __( 'Delete' ) }
-							onSecondaryActionClick={ () => onDelete( id ) }
-						/>
-					);
-				} ) }
-			</Fragment>
-		);
-	}
-}
+	return (
+		<>
+			<p>
+				{ __(
+					'Set up multiple ad units to use on your homepage, articles and other places throughout your site. You can place ads through our Newspack Ad Block in the Editor.'
+				) }
+			</p>
+			{ Object.values( adUnits ).map( ( { id, name } ) => {
+				return (
+					<ActionCard
+						key={ id }
+						title={ name }
+						actionText={ __( 'Edit' ) }
+						titleLink={ `#${ service }/${ id }` }
+						onClick={ () => history.push( `${ service }/${ id }` ) }
```

Comment:
Sidenote: a slightly different solution [is proposed here](https://github.com/Automattic/newspack-plugin/pull/936/files#diff-ebe9181a891d66ff9903af1ddc0652fcc33a11000450357f28e5a62b35410a44R73), where any internal links (relative links) are treated as such (not rendering an `ExternalLink` component).

Reply:
Oh, that's a good solution, too, and maybe a bit more DRY-esque. Should we merge this PR and move to your solution in #936, or do you want me to incorporate that here?

Reply:
Up to you, the result will be the same and the difference is small.

Reply:
I implemented in a9d47e5—I think it's a cleaner solution. Can you please re-test once more?